tests/unit/test_git_url_watcher.py

```diff
@@ -29,6 +29,17 @@
     run_command,
 )
 
+RETRYING_PARAMETERS: dict[str, Any] = {
+    "stop": stop_after_attempt(10),
+    "wait": wait_fixed(3),
+}
+
+
+def sleep_1_sec_to_make_commit_timestamp_unique():
```

Comment:
very minor final touch: put a ```_``` in front of the function. this shows it is a private function and only used in this module. 
same would be true of the ```RETRYING_PARAMETERS``` above.
but this is just informative, since this is a test module, this is not really necessary.

Reply:
will do thx